README.md

```diff
@@ -48,7 +48,7 @@
 ## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
[OPTIONAL] In the future, kindly remove the entire section for live demo if you were not instructed to deploy and you also. choose not to.